io/include/pcl/compression/impl/octree_pointcloud_compression.hpp

```diff
@@ -359,6 +359,12 @@ namespace pcl
           // decode differential color information
           std::vector<char>& pointDiffColorDataVector = color_coder_.getDifferentialDataVector ();
           compressed_tree_data_in_arg.read (reinterpret_cast<char*> (&point_diff_color_data_vector_size), sizeof (point_diff_color_data_vector_size));
+          PCL_CHECK_IO_STREAM(compressed_tree_data_in_arg, "size of pointDiffColorDataVector");
+
+          if (point_diff_color_data_vector_size < 0)
```

Comment:
Any reason for changing from `<= 0` to `< 0`?

Reply:
I checked the next lines and the functions work even with value 0, so I thought it would be preferred to let it run even on 0